projects/cdk/classes/assert.ts

```diff
@@ -18,12 +19,17 @@ interface DefaultView {
 }
 
 export function tuiAssertIsHTMLElement(node?: PossibleNode): asserts node is HTMLElement {
-    const document = (node as Node)?.ownerDocument;
-    const defaultView = document?.defaultView as unknown as DefaultView;
+    const defaultView = (node as Node)?.ownerDocument?.defaultView as unknown as
```

Comment:
I believe you can copy from 3.0 branch, this part is already fixed there

Reply:
fixed